fe/fe-core/src/main/java/com/starrocks/sql/analyzer/PolymorphicFunctionAnalyzer.java

```diff
@@ -334,7 +334,7 @@ public static Function generatePolymorphicFunction(Function fn, Type[] paramType
         if (!allRealElementType.isEmpty()) {
             Type commonType = allRealElementType.get(0);
             for (Type type : allRealElementType) {
-                commonType = getSuperType(commonType, type);
+                commonType = TypeManager.getCommonSuperType(commonType, type);
```

Comment:
L334-341
commonType = TypeManager.getCommonSuperType(allRealElementType) ?

Reply:
Sorry, the code is different when type can't match, `TypeManager.getCommonSuperType` will throw exception, but function pick need `return null`